docs/scripting-element-object.rst

```diff
@@ -0,0 +1,566 @@
+    - nodeType (Yes)
+    - nodeValue (No)
+    - ownerDocument (Yes)
+    - parentNode (Yes)
+    - parentElement (Yes)
+    - previousSibling (Yes)
+    - rootNode (Yes)
+    - textContent (No)
+
+The list of supported methods:
+
+Methods inherited from EventTarget_:
+    - addEventListener
+    - removeEventListener
+
+Methods inherited from HTMLElement_:
+    - blur
+    - click
+    - focus
+
+Methods inherited from Element_:
+    - getAttribute
+    - getAttributeNS
+    - getBoundingClientRect
+    - getClientRects
+    - getElementsByClassName
+    - getElementsByTagName
+    - getElementsByTagNameNS
+    - hasAttribute
+    - hasAttributeNS
+    - hasAttributes
+    - querySelector
+    - querySelectorAll
+    - releasePointerCapture
+    - remove
+    - removeAttribute
+    - removeAttributeNS
+    - requestFullscreen
+    - requestPointerLock
+    - scrollIntoView
+    - setAttribute
+    - setAttributeNS
+    - setPointerCapture
+
+Methods inherited from Node_:
+    - appendChild
+    - cloneNode
+    - compareDocumentPosition
+    - contains
+    - hasChildNodes
+    - insertBefore
+    - isDefaultNamespace
+    - isEqualNode
+    - isSameNode
+    - lookupPrefix
+    - lookupNamespaceURI
+    - normalize
+    - removeChild
+    - replaceChild
+
+Also, you can attach event handlers to the specified event. When the handler is called it will
+receive ``event`` table with the almost all available methods and properties.
+
+.. code-block:: lua
+
+    function main(splash)
+        local element = splash:select('.element')
+
+        local x, y = 0, 0
+
+        element.onclick = function(event)
+            event:preventDefault()
+            x = event.clientX
+            y = event.clientY
+        end
+
+        assert(splash:wait(10))
+
+        return x, y
+    end
+
+
+The following fields are read-only.
+
+.. _HTMLElement: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement
+.. _Element: https://developer.mozilla.org/en-US/docs/Web/API/Element
+.. _Node: https://developer.mozilla.org/en-US/docs/Web/API/Node
+.. _Event: https://developer.mozilla.org/en-US/docs/Web/API/Event
+.. _EventTarget: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget
+
+
+.. _splash-element-inner_id:
+
+element.inner_id
+----------------
+
+Id of the inner representation of the element.
+
+Methods
+~~~~~~~
+
+To modify or retrieve some information about the element you can use the following methods.
+Note that all of the following methods return flag indicating whether the operation was
+successful or not. The methods returns ``ok, reason_or_value`` pair. If ``ok`` is nil
+then error happened during the operation; ``reason`` provides an information about error type;
+otherwise ``ok`` is ``true`` and the returned value is stored in the second variable.
+
+
+.. _splash-element-exists:
+
+element:exists
+--------------
+
+Check whether the element exists in DOM. If the element doesn't exist some of the methods will fail raising
```

Comment:
It can disappear if some JS removes it from DOM. But it will be still present in memory. 
You can get non-existing elements using the selected element. E.g with `splash.node:clone()`